src/MPAGSCipher/CeaserCipher.cpp

```diff
@@ -0,0 +1,59 @@
+#include<string>
+#include<iostream>
+#include<vector>
+
+std::string CeaserCipher(std::string& inputText, const size_t key, bool encrypt){
+    //
+    std::string encryptedText{""};
+    int wrapShift{};
+    char inputChar{'x'};
+    size_t modKey = key % 26; //Ensure given key is between 0 - 26
+
+    //Define constant vector to store alphabet
+    const std::vector<char> alphabet{'A', 'B', 'C', 'D', 'E', 'F', 'G',
+                                            'H', 'I', 'J', 'K', 'L', 'M', 'N',
+                                            'O', 'P', 'Q', 'R', 'S', 'T', 'U',
+                                            'V', 'W', 'X', 'Y', 'Z'};
+    const std::size_t len{inputText.size()};
+
+    if (encrypt) { //Encryption Procedure
+        for (std::size_t i{0}; i < len; i++) {
+            inputChar = inputText[i];
+            for (std::size_t j{0}; j < 26; j++) {
+                wrapShift = (j+modKey) % 26;
+                if (inputChar == alphabet[j]) {
+                    encryptedText += alphabet[wrapShift];
+                }
+            }
+        }
+    } else { //Decryption Procedure
+        for (std::size_t i{0}; i < len; i++) {
+            inputChar = inputText[i];
+            for (std::size_t j{0}; j < 26; j++) {
+                wrapShift = (j+26-modKey) % 26; //Ensure wrapShift is positive before modulo is taken
+                if (inputChar == alphabet[j]) {
+                    encryptedText += alphabet[wrapShift];
+                }
+            }
+        }
+    }
```

Comment:
The majority of the code for decrypting and encrypting is the same so I would put the conditional at the point where you determine the shift. This would reduce copy+pasted code.